dialect/sql/builder.go

```diff
@@ -1061,6 +1061,42 @@ func (p *Predicate) ContainsFold(col, sub string) *Predicate {
 	})
 }
 
+func CompositeGT(columns []string, args ...interface{}) *Predicate {
+	return (&Predicate{}).CompositeGT(columns, args...)
+}
+
+func CompositeLT(columns []string, args ...interface{}) *Predicate {
+	return (&Predicate{}).CompositeLT(columns, args...)
+}
+
+// GT returns a composite ">" predicate.
+func (p *Predicate) CompositeGT(columns []string, args ...interface{}) *Predicate {
+	const operator = " > "
+	return p.append(func(b *Builder) {
+		b.Nested(func(nb *Builder) {
+			nb.IdentComma(columns...)
+		})
+		b.WriteString(operator)
+		b.WriteString("(")
+		b.Args(args...)
+		b.WriteString(")")
+	})
+}
+
+// LT appends a composite "<" predicate.
+func (p *Predicate) CompositeLT(columns []string, args ...interface{}) *Predicate {
+	const operator = " < "
+	return p.append(func(b *Builder) {
+		b.Nested(func(nb *Builder) {
+			nb.IdentComma(columns...)
+		})
+		b.WriteString(operator)
+		b.WriteString("(")
+		b.Args(args...)
+		b.WriteString(")")
+	})
+}
+
```

Comment:
Maybe, create a `compositeP` function (like you did in #215), and apply it with the different operator?